README.md

```diff
@@ -619,6 +619,26 @@ Running the `telegraf` agent will also clean up the previous metrics, so you are
 > [!Warning]
 > The metrics must be imported after each scenario run and cleanup, since they are associated only to the latest scenario run.
 
+### Import Holochain Metrics
+
+To import Holochain metrics into InfluxDB, holochain must be run with metrics enabled to output to file, and then use `telegraf`, a tool that collects and sends metrics to InfluxDB.
+
+To run Holochain with metrics, the `HOLOCHAIN_INFLUX_FILE` environment variable must be set. For example:
+```bash
+export HOLOCHAIN_INFLUX_FILE=telegraf/metrics/holochain.influx
+```
+
+Once you've finished running a scenario, you can start the `telegraf` agent to collect all metrics with
+
+```sh
+nix run .#local-telegraf
+```
+
+At this point the metrics will be imported to InfluxDB and you will be able to view the Holochain metrics in the InfluxDB Host dashboard by run_id.
```

Comment:
Is this right? The host dashboard?

On InfluxDB, the variable shows up as RunId -> https://github.com/holochain/wind-tunnel/blob/main/influx/templates/variables/runid.json#L10

Reply:
Changed to `InfluxDB dashboards`